docs/content/contributing/terraform/_index.md

```diff
@@ -61,7 +61,8 @@ The following tooling/extensions are recommended to assist you developing for th
 
 - [CodeTour extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour)
 - [Go (for writing tests)](https://go.dev/doc/install)
-- [Go extenstion for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=golang.go)
+- [Go extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=golang.go)
```

Comment:
```suggestion
- [Extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=golang.go)
```

Reply:
Should we propose extension?

Reply:
Likewise, for the bracketpair instructions below, the tip will only work if the matching extension is installed